src/app/grid/grid-sample-selection-template-numbers/grid-sample-selection-template-numbers.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="grid__wrapper">
+    <igx-grid #myGrid [data]="data" [primaryKey]="'ID'" [rowSelection]="'multiple'" [cellSelection]="'none'"
+        [paging]="true" [height]="'530px'" width="100%">
+        <igx-column [field]="'ContactName'" [width]="'20%'"></igx-column>
+        <igx-column [field]="'Country'" [width]="'20%'"></igx-column>
+        <igx-column [field]="'City'" [width]="'20%'" dataType="number"></igx-column>
+        <igx-column [field]="'PostalCode'" [width]="'20%'" dataType="number"></igx-column>
+        <igx-column [field]="'CompanyName'" [width]="'20%'" dataType="number"></igx-column>
+
+        <ng-template igxHeadSelector>
+            <div class="header-selector">
+                <img src="..\..\..\assets\images\card\avatars\igLogo.png" class="header-image">
+            </div>
+        </ng-template>
+
+        <ng-template igxRowSelector let-rowContext>
+            <div class="row-selector">
+                <div class="row-selector-index">
+                    <span>{{ rowContext.index + 1 }}</span>
+                </div>
+                <igx-checkbox class="row-selector-checkbox"
+                    [checked]="rowContext.selected"
+                    [readonly]="true"
+                    disableRipple="true" [disableTransitions]="myGrid.disableTransitions"
+                    [aria-label]="myGrid.rowCheckboxAriaLabel">
```

Comment:
Not sure what that is, but it's not a public prop:
![image](https://user-images.githubusercontent.com/3198469/64632884-a8e34300-d402-11e9-81db-a645c0be6ddb.png)